yml/3rd_party/acronis/qt5network.yml

```diff
@@ -0,0 +1,17 @@
+---
+Name: qt5network.dll
+Author: Micah Babinski
+Created: 2024-12-17
+Vendor: Acronis
+ExpectedLocations:
+  - '%PROGRAMFILES%\Acronis\CyberProtectHomeOffice'
```

Comment:
One more comment: if sideloading is used, aren't qt5network and syncagentsrv.exe expected to be located in the same location?